src/data/terraform.json

```
@@ -28,5 +29,77 @@
         "href": "https://support.hashicorp.com"
       }
     ]
-  }
+  },
+  "navigationHeaderItems": [
+    {
+      "label": "Overview",
+      "path": "/terraform",
+      "id": "overview"
+    },
+    {
+      "label": "CDKTF",
+      "path": "/terraform/cdktf",
+      "id": "cdktf"
+    },
+    {
+      "label": "CLI",
+      "path": "/terraform/cli",
+      "id": "cli"
+    },
+    {
+      "label": "Cloud",
+      "path": "/terraform/cloud-docs",
+      "id": "cloud-docs"
+    },
+    {
+      "label": "Config",
+      "path": "/terraform/config",
```

This 404s on `config`, I think you mean `configuration`?

```suggestion
      "path": "/terraform/configuration",
```